Apply Version Updates From Current Changes (#1054)
tauri-apps/wry: CHANGELOG.md

# Changelog

## \[0.35.0]

- [`dbea0f3`](https://github.com/tauri-apps/wry/commit/dbea0f3b1ff15dbb70902190336ba38b01a0e0f8)([#1071](https://github.com/tauri-apps/wry/pull/1071)) **Breaking change** Consistently use `WebView` in API names. The following APIs were renamed:

  - `WebviewExtWindows` → `WebViewExtWindows`
  - `WebviewExtUnix` → `WebViewExtUnix`
  - `WebviewExtMacOS` → `WebViewExtMacOS`
  - `WebviewExtIOS` → `WebViewExtIOS`
  - `WebviewExtAndroid` → `WebViewExtAndroid`
  - `WebviewUriLoader` → `WebViewUriLoader`
- [`dbea0f3`](https://github.com/tauri-apps/wry/commit/dbea0f3b1ff15dbb70902190336ba38b01a0e0f8)([#1071](https://github.com/tauri-apps/wry/pull/1071)) Add `WebViewExtWindows::set_memory_usage_level` API to set the [memory usage target level](https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2memoryusagetargetlevel) on Windows. Setting 'Low' memory usage target level when an application is going to inactive can significantly reduce the memory consumption. Please read the [guide for WebView2](https://github.com/MicrosoftEdge/WebView2Feedback/blob/main/specs/MemoryUsageTargetLevel.md) for more details.

> - [`dbea0f3`](https://github.com/tauri-apps/wry/commit/dbea0f3b1ff15dbb70902190336ba38b01a0e0f8)([#1071](https://github.com/tauri-apps/wry/pull/1071)) Refactor new method to take raw window handle instead. Following are APIs got affected:

Reviewer: These PR links are wrong because I edited the change files at #1071. Shall I create a PR to this PR branch for fixing those links?

Reviewer 2: Just add a review suggestion and I can update. Or I can still do this when it's about to publish

Reviewer: I added suggestions.

# Changelog

## \[0.35.0]

> - [`f41a8f0`](https://github.com/tauri-apps/wry/commit/f41a8f07eacbb60ebaad26a76d9e0f0ddc7adaf6)([#1070](https://github.com/tauri-apps/wry/pull/1070)) **Breaking change** Consistently use `WebView` in API names. The following APIs were renamed:

Reviewer: ```suggestion
- [`4f42acb`](https://github.com/tauri-apps/wry/commit/4f42acb4bd5ea7f16bf0ed18ea8113a9bd957659)([#1074](https://github.com/tauri-apps/wry/pull/1074)) **Breaking change** Consistently use `WebView` in API names. The following APIs were renamed:
```

  - `WebviewExtWindows` → `WebViewExtWindows`
  - `WebviewExtUnix` → `WebViewExtUnix`
  - `WebviewExtMacOS` → `WebViewExtMacOS`
  - `WebviewExtIOS` → `WebViewExtIOS`
  - `WebviewExtAndroid` → `WebViewExtAndroid`
  - `WebviewUriLoader` → `WebViewUriLoader`

> - [`f41a8f0`](https://github.com/tauri-apps/wry/commit/f41a8f07eacbb60ebaad26a76d9e0f0ddc7adaf6)([#1070](https://github.com/tauri-apps/wry/pull/1070)) Add `WebViewExtWindows::set_memory_usage_level` API to set the [memory usage target level](https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2memoryusagetargetlevel) on Windows. Setting 'Low' memory usage target level when an application is going to inactive can significantly reduce the memory consumption. Please read the [guide for WebView2](https://github.com/MicrosoftEdge/WebView2Feedback/blob/main/specs/MemoryUsageTargetLevel.md) for more details.

Reviewer: ```suggestion
- [`f420665`](https://github.com/tauri-apps/wry/commit/f42066576e1dfcaa8859bb76aa4d2e9859922e98)([#1053](https://github.com/tauri-apps/wry/pull/1053)) Add `WebViewExtWindows::set_memory_usage_level` API to set the [memory usage target level](https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2memoryusagetargetlevel) on Windows. Setting 'Low' memory usage target level when an application is going to inactive can significantly reduce the memory consumption. Please read the [guide for WebView2](https://github.com/MicrosoftEdge/WebView2Feedback/blob/main/specs/MemoryUsageTargetLevel.md) for more details.
```

> - [`f41a8f0`](https://github.com/tauri-apps/wry/commit/f41a8f07eacbb60ebaad26a76d9e0f0ddc7adaf6)([#1070](https://github.com/tauri-apps/wry/pull/1070)) Enhance initalization script implementation on Android supporting any kind of URL.

Reviewer: ```suggestion
- [`4d6f08e`](https://github.com/tauri-apps/wry/commit/4d6f08e8275378575d2c478c2162aac2934e5842)([#1076](https://github.com/tauri-apps/wry/pull/1076)) Enhance initalization script implementation on Android supporting any kind of URL.
```

> - [`f41a8f0`](https://github.com/tauri-apps/wry/commit/f41a8f07eacbb60ebaad26a76d9e0f0ddc7adaf6)([#1070](https://github.com/tauri-apps/wry/pull/1070)) Refactor new method to take raw window handle instead. Following are APIs got affected:

Reviewer: ```suggestion
- [`783b142`](https://github.com/tauri-apps/wry/commit/783b14239d367fb932c0d9236b60808ab0a6ec63)([#1041](https://github.com/tauri-apps/wry/pull/1041)) Refactor new method to take raw window handle instead. Following are APIs got affected:
```
